core/workspace_audio.js

```diff
@@ -11,10 +11,10 @@
  */
 'use strict';
 
-goog.provide('Blockly.WorkspaceAudio');
+goog.module('Blockly.WorkspaceAudio');
+goog.module.declareLegacyNamespace();
 
 goog.require('Blockly.internalConstants');
-goog.require('Blockly.utils');
```

Comment:
Just checking this was intentional.

Reply:
Yes. It's not used in this file and, as far as I can tell, this extra module doesn't have any side effects to require importing.